pages/_app.tsx

```diff
@@ -15,6 +14,7 @@ import DateFnsUtils from '@date-io/date-fns';
 
 import '../styles/globals.css';
 import { AuthProvider } from '../hooks/useAuth';
+import fullTheme from 'lib/material-ui/theme/fullTheme';
```

Comment:
Maybe only a personal preference but I am not a big fan of the naming and folder structure for the theme as we are introducing it in this PR.

I would prefer to have a file with the `baseTheme` that is called lib/material-ui/theme/baseTheme (currently it is in the file theme). And then have a file called `index` in which we bring together the `fullTheme` and the `baseTheme`. We could either do the "bringing together" directly in the `index` file or separately in the `fullTheme` file (as we have it now) and then import the `fullTheme` in `index`.

Then, when we import from the them folder, we just do `import theme from 'lib/material-ui/theme';` instead of `import fullTheme from 'lib/material-ui/theme/fullTheme';`

Reply:
I totally agree. I think the cleanest way is to use an index, export the default from the new theme file, and also the type Theme from the same file.